index.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>portfolio</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header>
+      <nav class="nav-container">
+        <div class="time-network">
+          <h4>9:41</h4>
+          <ul class="network-only">
+            <li class="network-only-item">
+              <img src="image/CellularConnection.png" alt="" />
+            </li>
+            <li class="network-only-item">
+              <img src="image/Wifi.svg" alt="" />
+            </li>
+            <li class="network-only-item">
+              <img src="image/Capacity.svg" alt="" />
+            </li>
+          </ul>
+        </div>
+        <div class="name-and-burger">
+          <h2>Yohannes Dag</h2>
+          <img src="image/Union.svg" class="burger" alt="none" />
+        </div>
+      </nav>
+      <div class="home-container">
+        <h1 class="greeting">
+          Hey There .<br />
+          I'm Alex
+        </h1>
+        <h2 class="job">I’am a Software Developer</h2>
```

Comment:
- [ ] Your hamburger menu is not matched with the Figma design.  Please try to match it with the size. :+1: 

- [ ] Kindly reduce the gap between "There" and "**.**" :+1: 

- [ ] The line with the `h2` tag is not aligned properly. Please work on the left alignment to match the Figma design :+1: 

Your design             |  Figma Design
:-------------------------:|:-------------------------:
![m3](https://user-images.githubusercontent.com/70483835/164492604-733a85b1-81cd-4ec2-8afc-5f3f6539a0dc.png) | ![m2](https://user-images.githubusercontent.com/70483835/164492747-9dbe2098-6ac8-4f21-8401-fd91d5c11885.png)